### PR TITLE
feat(unbound): migrate code

### PR DESCRIPTION
**Pull request by PierrickV**
| Before migration               | After migration                                                    |
| ------------------------------ | ------------------------------------------------------------------ |
| `dns.question.name: sekoia.io` | `dns.question.name: sekoia.io.` the FQDN contains a dot at the end |

### CODE REVIEW

File: Unbound/unbound/tests/domain.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "message": "2019-10-08T11:29:05+02:00 hostfoo unbound[17265]: [17265:13] info: 127.0.0.1 reachms.corp.com. A IN"
+  },
+  "expected": {
+    "message": "2019-10-08T11:29:05+02:00 hostfoo unbound[17265]: [17265:13] info: 127.0.0.1 reachms.corp.com. A IN",
+    "dns": {
+      "question": {
+        "class": "IN",
+        "name": "reachms.corp.com.",
```
Comment (squioc) [inline Unbound/unbound/tests/domain.json:10]: Can you see to remove the trailing dot in the domain-name?
```suggestion
        "name": "reachms.corp.com",
```

**Review by squioc — CHANGES_REQUESTED** [review]
The trailing dot stays in the domain name. I suggest removing it

File: Unbound/unbound/tests/domain.json (hunk 1 above)
Comment (PierrickV) [reply Unbound/unbound/tests/domain.json]: :ok_hand:

File: Unbound/unbound/ingest/parser.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          dns.question.class: "{{event.message.dns_question_class}}"
+          dns.question.name: "{{event.message.dns_question_name.strip('.')}}"
+          dns.question.type: "{{event.message.dns_question_type}}"
+          error.message: "{{event.message.error_message}}"
+          error.type: "{{event.message.error_type}}"
+          host.hostname: "{{event.message.hostname}}"
```
Comment (squioc) [inline Unbound/unbound/ingest/parser.yml:39]: I suggest using `host.name` instead:
```suggestion
          host.name: "{{event.message.hostname}}"
```

**Review by squioc — APPROVED** [review]
LGTM. Thanks for the work

File: Unbound/unbound/ingest/parser.yml (hunk 2 above)
Comment (PierrickV) [reply Unbound/unbound/ingest/parser.yml]: Done in commit https://github.com/SEKOIA-IO/intake-formats/pull/978/commits/5409491280de862f3f26905ae5364e33fca2fff0

**Review by squioc — DISMISSED** [review]

**Review by squioc — APPROVED** [review]

**Pull request merged**